tests/topotato/topotato/assertions.py

```diff
@@ -178,7 +178,7 @@ class AssertKernelRoutesV6(AssertKernelRoutes):
 
 
 class AssertVtysh(TopotatoAssertion, TimedMixin):
-    commands: OrderedDict[Tuple[str, str], List[Tuple[float, str, str, int, Any]]]
+    commands: "OrderedDict[Tuple[str, str], List[Tuple[float, str, str, int, Any]]]"
```

Comment:
Can you provide the error that this is fixing? I would prefer to address the actual issue over adding a workaround.

Reply:
Yes, there is the actual issue:

```sh
  File "/home/brunobernard/prod/frr/tests/topotato/topotato/__init__.py", line 11, in <module>
    from .assertions import *
  File "/home/brunobernard/prod/frr/tests/topotato/topotato/assertions.py", line 180, in <module>
    class AssertVtysh(TopotatoAssertion, TimedMixin):
  File "/home/brunobernard/prod/frr/tests/topotato/topotato/assertions.py", line 181, in AssertVtysh
    commands: OrderedDict[Tuple[str, str], List[Tuple[float, str, str, int, Any]]]
TypeError: 'type' object is not subscriptable
```

Reply:
Ah, ok. Not sure why this breaks, but it's not needed anymore since in python 3.6+ `OrderedDict` and `dict` is the same…

⇒ better to just remove OrderedDict

Reply:
https://github.com/opensourcerouting/frr/blob/6fe76bae9df59e789e9ab1f137d4f6f7ed389b52/tests/topotato/topotato/assertions.py#L16

Replacing `collections` with `typing` fixes the issue, since we are enforcing python 3.8. But I will remove the `OrderedDict` type hint from the code.

Reply:
fixed